### DISCUSSION TITLE
Misscomunications (GENERAL) [R]

### DISCUSSION THREAD

**Discussion by Michalx11** [secretcowlevel/DoomtrooperPublic#30] [Card Balance]
Very touchy subject I know but since it was discussed before how about we start by changing it to only cancel FATE cards

**Comment by framerate** [comment]
*watching intently*

**Comment by TomekPL-1** [comment]
If shifting 'Miscommunication' to Faction is an option I would suggest this:

Miscommunication (Cybertronic) [R]
EQUIPMENT, FACTION
Can only be given to UNIQUE warrior. Discard immediately after opponent plays a CONDITION, INFLUENCE, or Fate card. The card has no effect on play and is discarded/ANNIHILATED as normal. Any Actions or [DP] spent to play the card are lost.

**Comment by TomekPL-1** [comment]
Simplier is better:

Miscommunication (Cybertronic) [R]
EQUIPMENT, FACTION
Discard immediately after opponent plays a CONDITION, INFLUENCE, or Fate card. The card has no effect on play and is discarded/ANNIHILATED as normal. Any Actions or [DP] spent to play the card are lost.

**Reply by Sol-au-fein** [reply]
Thinking about it, with Gun Running or VAC engineer, you can equip it with 0 actions and it has no drawback compared to Young Guard, Kenji and Self-Sacrifice.

**Reply by TomekPL-1** [reply]
yeah - but on the other hand can be discarded by cards affecting equipment

**Reply by Sol-au-fein** [reply]
Malfunction or Wrath of Algeroth
You need Malfunctions to negate their bombs :/ so in the end it is not drawback, but insurance your bomb will blow or Slayer will slay.
